src/NHibernate.Test/SystemTransactions/AutoJoinSettingFixture.cs

```diff
@@ -0,0 +1,55 @@
+using System.Transactions;
+using NHibernate.Cfg;
+using NUnit.Framework;
+
+namespace NHibernate.Test.SystemTransactions
+{
+	[TestFixture(true)]
+	[TestFixture(false)]
+	[TestFixture(new object[] { null })]
+	public class AutoJoinSettingFixture : TestCase
+	{
+		private readonly bool? _autoJoinTransaction;
+
+		public AutoJoinSettingFixture(bool? autoJoinTransaction)
```

Comment:
You can use `[Values]` here: https://github.com/nunit/docs/wiki/Values-Attribute